assets/css/materialize.css

```diff
@@ -2262,6 +2262,7 @@ li.youtube {	list-style-image: url('/assets/images/youtube.svg'); }
 
 .black-text {
   color: #000000 !important;
+  font-size: medium;
```

Comment:
I’m concerned about this change. This class Is used in a number of places. I haven’t looked yet to see how those different places are affected.

Reply:
This shouldn't be done IMO. As this class is used in the nav section too, where on small screen there can be issues with the breaking of UI. And in other places too,. If you have checked all parys then, it's okay.

Reply:
Okay will verify it again and update if required !!